ui/packages/consul-peerings/app/components/consul/peer/form/generate/index.hbs

```diff
@@ -47,6 +47,19 @@
       </fsm.State>
 
       <fsm.State @matches={{'success'}}>
+        {{yield (hash
+          Fieldsets=(component "consul/peer/form/token/fieldsets"
+            item=@item
+            token=fsm.state.context.PeeringToken
+            regenerate=@regenerate
```

Comment:
I know this is already being used in this template but where are we passing this?

Reply:
Sometimes you want to "generate" a token, therefore you need to see a form to allow you to type the peers name, before seeing a token. Whereas sometimes you already have the peer, and you just need to 'regenerate' the token, therefore not needing to enter the name. This `@regenerate` boolean is just to say whether to immediately load a token, or show a form to ask for a name first.